docs/Building-Transmission.md

```diff
@@ -38,53 +38,35 @@ ninja -C build transmission-gtk
 
 ## On Unix ##
 ### Prerequisites ###
-#### Ubuntu ####
-On Ubuntu, you can install the required development tools with this command:
 
+#### Debian 11 / Bullseye ####
+On Debian, you can build transmission with a few dependencies on top of a base installation.
+
+For building transmission-daemon you will need basic dependencies
 ```console
-$ sudo apt-get install build-essential automake autoconf libtool pkg-config intltool libcurl4-openssl-dev libglib2.0-dev libevent-dev libminiupnpc-dev libgtk-3-dev libappindicator3-dev
+$ sudo apt install git build-essential cmake libcurl4-openssl-dev libssl-dev
 ```
+You likely want to install transmission as a native GUI application. There are two options, GTK and QT.
 
-_After you install those you can skip [to this section](#building-from-a-tarball)._
-
-#### Debian Squeeze ####
-Sometimes you have a need to stay current with upstream releases, even though you would like to rely on the stability of your base distribution. Here is how this can be accomplished in a "quick and dirty" fashion. Lines starting with a # have to be executed as root, lines starting with $ should be run as a regular user.
-
-1. Dependencies
-   First let us install every dependency Transmission needs and for which there is a usable version in the Debian repository.
-   ```console
-   # apt-get install ca-certificates libcurl4-openssl-dev libssl-dev pkg-config build-essential checkinstall
-   ```
-
-2. libevent
-   Traditionally libevent is also needed, but Transmission depends on version numbers only rarely found in Debian. So let us start by compiling libevent in a directory of your choice. Browse to https://libevent.org/ and get the latest version.
-   ```console
-   $ cd /var/tmp
-   $ wget https://github.com/downloads/libevent/libevent/libevent-2.0.18-stable.tar.gz
-   $ tar xf libevent-2.0.18-stable.tar.gz
-   $ cd libevent-2.0.18-stable
-   $ CFLAGS="-Os -march=native" ./configure && make
-   ```
-
-   Now, we would really like to be able to upgrade to a new version in the future, so there should be a mechanism other than the classic "make install" which keeps count of what went where (and ideally this is not a piece of paper). So we build a very simple Debian package from the compiled files and install it. Basically you just enter the following command and hit return until a nice text message tells you that all is done.
+For GTK 3 client, two additional packages are required
+```console
+$ sudo apt install libgtkmm-3.0-dev gettext
+```
```

Comment:
If you know it, can you also document GTK 4 client installation?

Reply:
GTK 4 is currently not in Debian stable repositories, which means it's not possible to install it on Debian 11.
It might be possible on Debian 12 when it's ready in about 6 months or so, but those who use Debian unstable should be able to adapt these instructions for their constantly changing environment.